app/utils/get_filtered_records.py

```diff
@@ -18,25 +18,25 @@ def _get_isotype_col_expression(label:str = "isotypes"):
                 [
                     (and_(DashboardSource.isotype_igg == 'true',
                           DashboardSource.isotype_igm == 'true',
-                          DashboardSource.isotype_iga == 'true'), array(['IgG, IgM, IgA'])),
```

Comment:
I wonder whether there's a better way to do this... 

if DashboardSource.isotype_igx: array.append('igx')
the 8x arrays doesn't seem like the best of design patterns to me!!

this is less of a comment on this PR and more of a general comment